src/plugins/intel_gna/src/gna_graph_compiler.cpp

```diff
@@ -361,14 +388,15 @@ void GNAGraphCompiler::ConvolutionPrimitive(InferenceEngine::CNNLayerPtr layer)
     auto in_kernel_h = convolution._kernel_y;
     bool transpose_h_w = false;
 
-    // Map 2d convolution to 1d if it's possible.
-    if (!ShouldUseOnlyConv2DGnaIface() && gna_convolution_layer::isMappableFrom2DTo1D(in_height,
-                                                                                      in_width,
-                                                                                      in_channels,
-                                                                                      convolution._kernel_y,
-                                                                                      convolution._kernel_x,
-                                                                                      convolution._stride_y,
-                                                                                      convolution._stride_x)) {
+    // Map 2d convolution to 1d if it's possible
+    if (!is_dwsc && !ShouldUseOnlyConv2DGnaIface() &&
```

Comment:
Can we have situation that ShouldUseOnlyConv2DGnaIface() will return false but is_dwsc will be true?

Reply:
is_dwsc check can be removed without changing the logic, because it's broke anyway for group convolutions.